Jint/Runtime/Debugger/BreakLocation.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Jint.Runtime.Debugger
```

Comment:
could use file-scoped namespaces for new files and set #nullable enable to reduce later work